src/identifier.h

```diff
@@ -21,12 +21,18 @@
 
 class Identifier : public RootObject
 {
+private:
```

Comment:
> That suggests there's a member function lacking. What would that be? I've purged all these direct accesses from dmd. (How is GDC bypassing the private: anyway?)

Still using the 2.068 implementation, and it's not private in that version.

I'm backporting changes to C++ that are relevant to priming ourselves for switching to D - using the same headers as upstream is the first tactic being used.

This is being done because I don't want to maintain 2.068 and whatever the next version we'll support in two separate branches.